package.json

```diff
@@ -58,15 +58,15 @@
     "regenerator-runtime": "0.13.9",
     "timeago.js": "4.0.2",
     "tinymce": "5.10.2",
-    "yup": "0.32.11"
+    "yup": "0.31.1"
```

Comment:
[Downgraded](https://github.com/edx/frontend-app-discussions/pull/37/commits/05db3a168d64b33d00497bff35d96de16c15a67b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R61)?

Reply:
Yes, it seems that the newer version of yup causes certain lint checks to fail. We can upgrade once the code has moved to Node 16 and support for IE11 has been dropped.